index.js

```diff
@@ -244,6 +257,33 @@ class MarkdownToolbarElement extends HTMLElement {
   }
 }
 
+const focusListeners = new WeakMap()
+
+function focusKeydown(toolbar: MarkdownToolbarElement, event: KeyboardEvent) {
+  const key = event.key
+  if (key !== 'ArrowRight' && key !== 'ArrowLeft' && key !== 'Home' && key !== 'End') return
+  const target = event.target
+  if (!(target instanceof HTMLElement)) return
+  if (!target.hasAttribute('data-md-button')) return
+  if (target.closest('markdown-toolbar') !== toolbar) return
+
+  const buttons = []
+  for (const button of toolbar.querySelectorAll('[data-md-button]')) {
+    if (!(button instanceof MarkdownButtonElement)) continue
```

Comment:
This would reject all custom buttons.

Could you add a test for custom buttons as well?

```suggestion
```

Reply:
Missed this one in a rebase.